code/modules/events/event_manager.dm

```diff
@@ -1,7 +1,7 @@
 var/list/allEvents = typesof(/datum/event) - /datum/event
 
-var/eventTimeLower = 6000	//10 minutes
```

Comment:
undocumented change, you said in the OP it was 15 minutes

Reply:
`roundstart_delay` is the variable that I am referring to, `eventTimeLower` and `eventTimeUpper` refer to the time between event triggers, which I referred to as "spread", but yeah I can make that more clear